lahja/endpoint.py

```diff
@@ -258,10 +258,13 @@ def connect_to_endpoints_nowait(self, *endpoints: ConnectionConfig) -> None:
         self._connect_to_endpoint(endpoint)
 
     def _connect_to_endpoint(self, endpoint: ConnectionConfig) -> None:
```

Comment:
Maybe naming this `_connect_if_not_already_connected` would make this functionality more explicitly clear when reading the code in places where it's used.

Reply:
Sounds good to me!